doc/design/linc_permissions.md

```diff
@@ -0,0 +1,54 @@
+# Permissions for S3 Assets in LINC Archive
+
+Collaborators: Aaron Kanzer, Kabilar Gunalan
+
+## Requirements
+
+• As a CONNECTS developer, I shall be able to view multiple datasets/layers (e.g. dMRI, tractography, HiP-CT) in the same neuroglancer
+• As a CONNECTS developer, I shall be able to view data locally or on neuroglancer
+• As a CONNECTS developer, I shall be able to have direct access to S3 asset URI
+• As a LINC Archive user, I shall be able to render neuroglancer layers view similar to DANDI
+
+#### Relevant Links
+
+https://github.com/orgs/lincbrain/projects/2?pane=issue&itemId=54650571
+
+https://github.com/neuroscales/ngtools/blob/main/notebooks/show_tract.ipynb
+
+https://github.com/google/neuroglancer/issues/507
+
+## Implementation Options
+
+#### Issue read-only credentials for S3 bucket. Quick set up.
+
+Does not resolve rendering issue. Would resolve accessing the asset in Jupyter Notebook, but would require flaky Python code.
+
+#### Integrate read-only credentials for S3 bucket into LINC Archive, LINC CLI
+
+###### CloudFront distribution with Origin Access Identity control for relevant S3 buckets
+
```

Comment:
<img width="946" alt="image" src="https://github.com/lincbrain/linc-archive/assets/871137/f7e79ed0-803e-4567-b0a1-c2ba78a188ff">

I converted your version of the drawio diagram shown above to a mermaid diagram suggested below so that we can integrate it in this design doc.

```mermaid
%%{init: {"flowchart": {"curve": "linear"}}}%%

flowchart LR
    E --> B
    A(User) -->| If client has CloudFront cookies from prior session, <br/> then proceed. | B(Static Webpage i.e. Neuroglancer)
    A --> | If client does not have CloudFront cookies, <br/> then GET upon /api/permissions/s3 in LINC Archive API. | E(LINC Archive API)
    B --> | Upon user activity, <br/> sends presigned cookies | C(AWS CloudFront)
    C -->| Allows data to be fetched | D(Private AWS S3 Bucket)
    D -->| 1. Neuroglancer able to access S3 data <br/> 2. Data rendered on screen | B
```